openapi.yaml

```diff
@@ -976,6 +1052,7 @@ paths:
         - url: 'https://global.api.konghq.com/'
       tags:
         - Networks
+      x-speakeasy-group: CloudGateways
```

Comment:
This might have caused the additional changes to some resources - is this needed?